bin/handler-microsoft-teams.rb

```diff
@@ -176,11 +176,11 @@ def payload(notice)
         text: [teams_message_prefix, notice].compact.join(' ')
       }],
       potentialAction: [{
-        '@type': teams_action_type ? teams_action_type : "OpenUri",
-        name: teams_action_name ? teams_action_name : "View in Sensu",
+        '@type' => teams_action_type ? teams_action_type : 'OpenUri',
```

Comment:
I am not quite sure why the key with an `@` needed `=>` over `:` but I am willing to trust the rubocop gods on this one. Would be curious if someone could tell me why it is special.